### ISSUE TITLE
php-xdebug requires php7.0 even though php7.1 is installed

### ISSUE THREAD

**Issue by luispabon** [oerdnj/deb.sury.org#640]
Seems to have happened after 7.1.7 was added to the repo.

Below `dist-upgrade` was after I manually updated 7.1.6 > 7.1.7 while trying to track down which package was trying to bring in 7.0

```
 ~  sudo apt dist-upgrade  --no-install-recommends                                                                                                                Thu 06 Jul 2017 18:25:10 BST
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Calculating upgrade... Done
The following package was automatically installed and is no longer required:
  libssl1.0.2
Use 'sudo apt autoremove' to remove it.
The following NEW packages will be installed
  php7.0-cli php7.0-common php7.0-json php7.0-opcache php7.0-phpdbg php7.0-readline
The following packages will be upgraded:
  php-xdebug
1 to upgrade, 6 to newly install, 0 to remove and 0 not to upgrade.
Need to get 0 B/4,206 kB of archives.
After this operation, 12.7 MB of additional disk space will be used.
Do you want to continue? [Y/n] 
```

```
 ~  apt list|grep php|grep installed                                                                                                997ms  Thu 06 Jul 2017 18:25:47 BST

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

php-apcu/zesty,now 5.1.8+4.0.11-1+deb.sury.org~zesty+1 amd64 [installed]
php-apcu-bc/zesty,now 1.0.3-5+deb.sury.org~zesty+1 amd64 [installed,automatic]
php-codesniffer/zesty,zesty,now 2.7.1-1 all [installed]
php-common/zesty,zesty,now 1:53~ubuntu17.04.1+deb.sury.org+1 all [installed]
php-igbinary/zesty,now 2.0.1-1+deb.sury.org~zesty+1 amd64 [installed,automatic]
php-libsodium/zesty,now 1.0.6-1+deb.sury.org~zesty+1 amd64 [installed]
php-redis/zesty,now 3.1.2-1+deb.sury.org~zesty+1 amd64 [installed]
php7.1-bz2/zesty,now 7.1.7-1~ubuntu17.04.1+deb.sury.org+1 amd64 [installed]
php7.1-cli/zesty,now 7.1.7-1~ubuntu17.04.1+deb.sury.org+1 amd64 [installed]
php7.1-common/zesty,now 7.1.7-1~ubuntu17.04.1+deb.sury.org+1 amd64 [installed,automatic]
php7.1-curl/zesty,now 7.1.7-1~ubuntu17.04.1+deb.sury.org+1 amd64 [installed]
php7.1-gd/zesty,now 7.1.7-1~ubuntu17.04.1+deb.sury.org+1 amd64 [installed]
php7.1-intl/zesty,now 7.1.7-1~ubuntu17.04.1+deb.sury.org+1 amd64 [installed]
php7.1-json/zesty,now 7.1.7-1~ubuntu17.04.1+deb.sury.org+1 amd64 [installed,automatic]
php7.1-mbstring/zesty,now 7.1.7-1~ubuntu17.04.1+deb.sury.org+1 amd64 [installed]
php7.1-mysql/zesty,now 7.1.7-1~ubuntu17.04.1+deb.sury.org+1 amd64 [installed]
php7.1-opcache/zesty,now 7.1.7-1~ubuntu17.04.1+deb.sury.org+1 amd64 [installed,automatic]
php7.1-pgsql/zesty,now 7.1.7-1~ubuntu17.04.1+deb.sury.org+1 amd64 [installed]
php7.1-readline/zesty,now 7.1.7-1~ubuntu17.04.1+deb.sury.org+1 amd64 [installed,automatic]
php7.1-sqlite3/zesty,now 7.1.7-1~ubuntu17.04.1+deb.sury.org+1 amd64 [installed]
php7.1-xml/zesty,now 7.1.7-1~ubuntu17.04.1+deb.sury.org+1 amd64 [installed]
php7.1-xsl/zesty,zesty,now 7.1.7-1~ubuntu17.04.1+deb.sury.org+1 all [installed]
php7.1-zip/zesty,now 7.1.7-1~ubuntu17.04.1+deb.sury.org+1 amd64 [installed]
```
```
 ~  lsb_release -a                                                                                                                  977ms  Thu 06 Jul 2017 18:25:48 BST
LSB Version:	core-9.20160110ubuntu5-amd64:core-9.20160110ubuntu5-noarch:security-9.20160110ubuntu5-amd64:security-9.20160110ubuntu5-noarch
Distributor ID:	Ubuntu
Description:	Ubuntu 17.04
Release:	17.04
Codename:	zesty
```

**Comment by oerdnj** [comment]
Should be fixed in `2.5.5-3` version I have just uploaded. xdebug explodes when used with PHP 7.2, and I made a mistake limiting xdebug builds to 5.6-7.1

**Comment by luispabon** [comment]
:+1: can confirm it's fixed, thank you Ondrej
